index.html

```diff
@@ -7,14 +7,181 @@
+        </div>
+    </section>
+   <!-- FORM -->
+<section>
+    <div class="container">
+        <div class="row">
+            <div class="col-12 md">
+                <div class="request">
+                    <h4 class="form-title">Contact Form</h4>
 
-    <!-- Add your markup here -->
+                    <div class="form-container">
+                      <form action="/action_page.php">
+                        <label for="fname">First Name</label>
+                        <input type="text" id="fname" name="firstname" placeholder="Your name..">
+                    
+                        <label for="lname">Last Name</label>
+                        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+                    
+                        <label for="country">Country</label>
+                        <select id="country" name="country">
+                          <option value="australia">Australia</option>
+                          <option value="canada">Canada</option>
+                          <option value="usa">USA</option>
+                        </select>
+                    
+                        <label for="subject">Subject</label>
+                        <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+                    
+                        <input type="submit" value="Submit">
+                      </form>
+                    </div>
+                </div>
+            </div>
+        
+   <!-- MAP -->        
+            <div class="row-3">
+                <div class="col-12 md-col-6">
+                    <div class="card">
+                        <img src="/images/map.PNG" alt="hand">
+                    </div>   
+                </div> 
+                <div class="col-12 md-col-6"> 
+                <div class="row">
+          
+                        <div class="col-6">
+                            <h4>Address: </h4><h5> <br> Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis sunt placeat at odit, architecto corrupti nemo molestiae! Modi vero eius amet. Laborum assumenda, ullam beatae facere ea enim harum voluptatum.</h5>
+                        </div>
+                    <div class="col-6">
+                        <h5>Lorem ipsum dolor sit amet.</h5>
+                    </div>
+                    <div class="col-6">
+                        <hr><h4>Address: </h4><h5><br> Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis sunt placeat at odit, architecto corrupti nemo molestiae! Modi vero eius amet. Laborum assumenda, ullam beatae facere ea enim harum voluptatum.</h5>
+                    </div>
+                        <div class="col-6">
+                            <h4>jdf lorem10 dsfd df sdf dfdsf dfs sdf s s sdf sdfs fsd fs </h45>
+                        </div>
+                    </div>
+                </div>
+            </div>
+    </div>
+</section>
 
+<section>
```

Comment:
Instead of making a new section, this should be a `footer` instead

Reply:
you can then chain the footer element with mutiple classes, so the "footer" and "container" class can be applied to one html element